.github/workflows/spark_test.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
           pipenv run pip install flake8==3.5.0 pypandoc==1.3.3
           pipenv run pip install black==23.9.1
           pipenv run pip install importlib_metadata==3.10.0
-          pipenv run pip install mypy==0.982
+          pipenv run pip install mypy==1.8.0
```

Comment:
The mypy versions 0.982 and 1.8.0 have conflicting rules (cannot get the style check to pass with both versions on the same file) so we upgrade both